finance/management/commands/xformnetfilerawdata.py

```diff
@@ -273,7 +273,7 @@ def load_f460A_row(row, agency, verbosity=1):  # noqa
     beneficiary.save()
 
     # Now we have all the parts, save!
-    money = models.IndependentMoney(  # or calculated_Amount
+    money, _ = models.IndependentMoney.objects.get_or_create(  # or calculated_Amount
```

Comment:
It appears like the nefile data has already resolved revised filings as I do not see a column that indicates the filing revision number.